<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransliterationTab</class>
 <widget class="QWidget" name="TransliterationTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translitération</string>
  </property>
  <property name="locale">
   <locale language="French" country="France"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="transliterationSourceGroupBox">
     <property name="title">
      <string>Source</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="transliterationSourceFileLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="transliterationSourceInputRadioButton">
        <property name="text">
         <string>Texte</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="transliterationSourceFileRadioButton">
        <property name="text">
         <string>Fichier</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="transliterationSourceFilePushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Parcourir</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="transliterationTextSourcePlainTextEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="transliterationSourceTextSelectPushButton">
        <property name="text">
         <string>Sélectionner</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="transliterationConvertPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Convertir</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="transliterationResultGroupBox">
     <property name="title">
      <string>Résultat</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="transliterationTextResultPlainTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="transliterationResultFilePushButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Enregistrer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>transliterationSourceInputRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>transliterationTextSourcePlainTextEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transliterationSourceFileRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>transliterationSourceFileLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transliterationSourceFileRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>transliterationSourceFilePushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>722</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transliterationSourceInputRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>transliterationSourceTextSelectPushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
